ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,112 @@
+# Script used for checking changes for incremental testing cases
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+
+
+def list_changed_files(commit_range):
+    """Returns a list of names of files changed in the given commit range.
+
+    The function works by opening a subprocess and running git. If an error
+    occurs while running git, a RuntimeError will be raised.
+
+    Args:
+        commit_range (string): The commit range to diff, consisting of the two
+            commit IDs separated by \"..\"
+
+    Returns:
+        list: List of changed files within the commit range
+    """
+
+    command = ["git", "diff", "--name-only", commit_range]
+    proc = subprocess.Popen(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    out, err = proc.communicate()
+    if proc.returncode != 0:
+        raise RuntimeError("Command {cmdline} failed with code {returncode}, "
+                           "stderr was:\n{stderr}\n".format(
+                               cmdline=command,
+                               returncode=proc.returncode,
+                               stderr=err.decode()))
+
+    return list(filter(None, (s.strip() for s in out.decode().splitlines())))
+
+
+if __name__ == "__main__":
+
+    RAY_CI_TUNE_AFFECTED = "0"
```

Comment:
We can actually do `RAY_CI_TUNE_AFFECTED = 0` instead. I think that's a little cleaner.